src/Codeception/Command/Shared/ActorTrait.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Codeception\Command\Shared;
+
+trait ActorTrait
+{
+    protected function getActor(): ?string
```

Comment:
I would expect this function to actually return an Actor
Can we rename it to `getActorClassName`